src/components/grapher/GrapherDots.vue

```diff
@@ -1,13 +1,17 @@
 <template>
   <g class="grapher-dots--dots-container">
-    <circle
+    <Dot
       v-for="dot in stuntSheetDots"
       :key="`${dot.x}-${dot.y}-dots--dot`"
       class="grapher-dots--dot"
+      data-test="grapher-dots--dot"
+      :transform="`translate(${dot.xAtBeat(beat - 1)}, ${dot.yAtBeat(
+        beat - 1
+      )})`"
```

Comment:
I was thinking having a translation so that in the future if we want to show the dot in-line with other text (for example, showing dot type 1's dot appearance next to the description of dot type 1's continuities, here: 
<img width="237" alt="image" src="https://user-images.githubusercontent.com/29964426/101452036-f9856c80-38e1-11eb-9ce1-48bdab3774b4.png">
) we don't have to calculate x and y, we can just pop it in.

Reply:
Ah, I see. We should actually include the dot label in `Dot.vue`... What do you think?

Reply:
I don't think so because dots of the same type can have different labels

Reply:
Whoops I misinterpreted! Yes, that sounds like a good idea...

Reply:
Since the dot label is in Dot.vue, what do you think about the original idea of giving the Dot.vue an x and y prop, and the Dot transforms itself?